README.md

```diff
@@ -1,65 +1,52 @@
 
-Open [http://localhost:3000](http://localhost:3000) with your browser to see the result.
-
-You can start editing the page by modifying `pages/index.js`. The page auto-updates as you edit the file.
-
-[API routes](https://nextjs.org/docs/api-routes/introduction) can be accessed on [http://localhost:3000/api/hello](http://localhost:3000/api/hello). This endpoint can be edited in `pages/api/hello.js`.
+> You'll need your API key before making your first request!
+> **[Get Your Key Here](https://docs.codat.io/reference/authentication).**
 
-The `pages/api` directory is mapped to `/api/*`. Files in this directory are treated as [API routes](https://nextjs.org/docs/api-routes/introduction) instead of React pages.
+### Running on Vercel
 
-## Deploy on Vercel
+Hit the **Deploy** above to clone the project and start building!
 
-The easiest way to deploy your Next.js app is to use the [Vercel Platform](https://vercel.com/new?utm_medium=default-template&filter=next.js&utm_source=create-next-app&utm_campaign=create-next-app-readme) from the creators of Next.js.
+> You'll need your API key to initialize the project!
+> **[Get Your Key Here](https://docs.codat.io/reference/authentication).**
 
-Check out our [Next.js deployment documentation](https://nextjs.org/docs/deployment) for more details.
+## Next Steps
 
-[![Deploy with Vercel](https://vercel.com/button)](https://vercel.com/new/clone?repository-url=https%3A%2F%2Fgithub.com%2Fmcclowes%2Fcodat-nextjs-sample-project)
+### Learn more about Codat
 
-## Learn more about Codat
-
-To learn more about Codat and it's API, take a look at the following resources:
 - [Read our getting started guide for developers](https://docs.codat.io/docs/get-started-api)
 - [Read how to get authenticated](https://docs.codat.io/reference/authentication)
 - [Learn about our /comapnies endpoint](https://docs.codat.io/reference/listcompanies)
 
-## Learn more about Next.js
+### Learn more about Next.js
 
 To learn more about Next.js, take a look at the following resources:
 
 - [Next.js Documentation](https://nextjs.org/docs) - learn about Next.js features and API.
 - [Learn Next.js](https://nextjs.org/learn) - an interactive Next.js tutorial.
 
 You can check out [the Next.js GitHub repository](https://github.com/vercel/next.js/) - your feedback and contributions are welcome!
+
+## What is Codat?
+
+Codat is the universal API for small business data. We provide real-time connectivity to enable software providers and financial institutions to build integrated products for their small business customers.
+
+Small businesses run on many different platforms. Codat gives these platforms a standard way to connect with each other so they can move data around seamlessly at scale.
+
+[deploywithvercel]: https://vercel.com/new/clone?repository-url=https%3A%2F%2Fgithub.com%2Fmcclowes%2Fcodat-nextjs-sample-project&env=CODAT_API_KEY&envDescription=Your%20Codat%20API%20Key&envLink=https%3A%2F%2Fdocs.codat.io%2Freference%2Fauthentication
```

Comment:
Should this be here still?

Reply:
yes, this is the definition for the URL that's used in multiple places above

Reply:
Ohhhh... Didn't know markdown had vars like that!!!

Reply:
Not really vars, just for writing links with a shorter syntax, good it one reuses the same url multiple times